computedfields/management/commands/checkdata.py

```diff
@@ -72,7 +72,8 @@ def handle(self, *app_labels, **options):
         end_time = time()
         duration = int(end_time - start_time)
         self.eprint(f'\nTotal check time: {timedelta(seconds=duration)}')
-        sys.exit(1 if has_desync else 0)
+        if has_desync:
+            raise CommandError('Desync data found.', returncode=1)
```

Comment:
How does that CommandError behave in call_command? Will it raise there or just return the non 0 returncode? (Not sure atm, if call_command uses a subprocess...)